### PR TITLE
Adding oc_process to lib_openshift.

### PR DESCRIPTION
**Pull request by kwoodson**

### CODE REVIEW

File: roles/lib_openshift/src/doc/process [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+module: oc_process
+short_description: Module to process openshift templates
+description:
+  - Process openshift templates programmatically.
+options:
+  state:
+    description:
+    - Currently present is only supported state.
+    default: present
+    choices: ["present", "absent", "list"]
```
Comment (abutcher) [inline roles/lib_openshift/src/doc/process]: Docs: Is this accurate? It looks like `present` and `list` states are supported.

File: roles/lib_openshift/src/doc/process [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    description:
+    - A list of parameters that will be inserted into the template.
+    required: false
+    default: None
+    aliases: []
+  create:
+    description:
+    - Whether or not to create the template after being processed. e.q  oc process | oc create -f -
+    required: false
+    default: None
```
Comment (abutcher) [inline roles/lib_openshift/src/doc/process]: Docs: create defaults to false.

**Comment by kwoodson** [comment]
@abutcher @tbielawa,

I updated the docs and fixed the tests for the rebase that happened.

**Comment by ingvagabund** [comment]
No tests run? This breaks the deployment: https://aos-ci.s3.amazonaws.com/openshift/openshift-ansible/jenkins-openshift-ansible-3-test-matrix-CONTAINERIZED=_NOT_containerized,OSE_VER=3.4,PYTHON=System-CPython-2.7,TOPOLOGY=openshift-cluster,TargetBranch=master,nodes=openshift-ansible-slave-867/75febd192b62072220b56eca1b2d93d9f085b4c7.txt

File: roles/lib_openshift/tasks/main.yml [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
-    name: python-ruamel-yaml
+    name: "{{ item }}"
     state: present
+  with_items:
+  - ruamel.yaml
+  - ruamel.yaml
+  - ruamel.yaml
+  - ruamel.yaml
+  - ruamel.yaml
+  - ruamel.yaml
```
Comment (ingvagabund) [inline roles/lib_openshift/tasks/main.yml]: How this happened? Six times the same item? ``ruamel.yaml`` is not a package name. What was wrong with ``python-ruamel-yaml``?

**Comment by ingvagabund** [comment]
Koji: python-ruamel-yaml
Brew: python-ruamel-yaml
CentOS: No build

This eliminates CentOS deployments unless there is some hidden repository that provides the rpm.

**Comment by ingvagabund** [comment]
Reverting the PR until the rpms and the ``ruamel.yaml`` package name is resolved.

**Pull request merged**